src/views/components/dashboard/charts/chart-slow.vue

```diff
@@ -22,7 +22,12 @@ limitations under the License. -->
         </svg>
         <div class="mb-5 ell" v-tooltip:top.ellipsis="i.name || ''">
           <span class="calls sm mr-10">{{ i.value }}</span>
-          <span class="cp link-hover">{{ i.name + getTraceId(i) }}</span>
+          <span class="cp link-hover">
+            <a v-if="i.url" :href="i.url" target="_blank">
```

Comment:
You should use `router-link`.

```suggestion
      <router-link :to="{ name: 'trace',  query: {... } }">...</router-link>
```

Reply:
I fixed it.